_includes/fix_analytics_signup_form.html

```diff
@@ -0,0 +1,352 @@
+                return;
+            }
+
+            showOverlay();
+
+            var customer = {
+                email: emailField.value,
+                currency: currencyField.value,
+                locale: localeField.value,
+                type: customerTypeField.value,
+                name: nameField.value,
+                countryId: locationCountryIdField.value,
+                street: locationStreetField.value,
+                cityText: locationCityField.value,
+                stateText: locationStateField.value,
+                zipcode: locationZipField.value,
+                phone: phoneField.value,
+                registrationNo: registrationNumberField.value,
+            };
+            var domainList = document.querySelectorAll("[data-domain-field]");
+            var domains = [];
+            var domainValues = domainList.forEach((input) => {
+                if (input.value) {
+                    domains.push(input.value);
+                }
+            });
+            var voucher = voucherField.value;
+
+            var data = {
+                customer,
+                domains,
+                voucher,
+            };
+
+            sendRequestToApi(form.action, data);
+
+            return false;
+        });
+
+        async function sendRequestToApi(url, data) {
+            try {
+                const rawResponse = await fetch(url, {
+                    method: "POST",
+                    headers: {
+                        Accept: "application/json",
+                        "Content-Type": "application/json",
+                    },
+                    body: JSON.stringify(data),
+                });
+                const content = await rawResponse.json();
+
+                console.log("success", content);
+                showSuccess();
+            } catch (error) {
+                console.log("error", error);
+                showFailure();
+            }
+        }
+
+        function addNewDomainField() {
+            var input = document.createElement("INPUT");
+            input.setAttribute("Type", "text");
+            input.setAttribute("Placeholder", "www.your-domain.com");
+            input.setAttribute("data-domain-field", true);
+            domainsContainer.appendChild(document.createElement("label")).appendChild(input);
+        }
+
+        function showOverlay() {
+            contactFormOverlay.innerText = "Getting in touch...";
+            contactFormOverlay.classList.remove("contact-form-overlay-clickthrough");
+            requestAnimationFrame(function () {
+                contactFormOverlay.classList.add("contact-form-overlay-visible");
+            });
+        }
+
+        function showSuccess() {
+            // TODO: Make the success state better/clearer
+            contactFormOverlay.classList.remove("contact-form-overlay-visible");
+
+            setTimeout(function () {
+                contactFormOverlay.innerText = "Got in touch! 👍";
```

Comment:
Better text here to confirm you have gotten an email and your order has been registered and you can now use the tool

Reply:
I have updated the overlay messages (they are also translatable now).